## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 466553`
Commit: [`c368e7a0c17782e1f96ba957ebd8df15181fb590`](https://github.com/NixOS/nixpkgs/commit/c368e7a0c17782e1f96ba957ebd8df15181fb590) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/c368e7a0c17782e1f96ba957ebd8df15181fb590..pull/466553/head))
Merge: [`8b8f5492c93c26880f1df391c1a7d615c3fbfed8`](https://github.com/NixOS/nixpkgs/commit/8b8f5492c93c26880f1df391c1a7d615c3fbfed8)

Logs: https://github.com/acid-bong/nixpkgs-review-gha/actions/runs/19798980836

<details><summary>Download packages from cache:</summary><ul>
<li><details><summary><code>x86_64-linux</code></summary>

```shell
nix-store -r --add-root nixpkgs-pr-466553-x86_64-linux \
  --option binary-caches 'https://cache.nixos.org/ https://acidbong.cachix.org' \
  --option trusted-public-keys '
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  acidbong.cachix.org-1:ycS7jRCupEvvQRTlKcB4mjtCYUSlhioNQNwmrPs3E84=
  ' \
  /nix/store/40p3va1r3if1qfyp7xlk8n1ysdfi4gyx-pinsel-0-unstable-2022-03-27
```
</details></li>
<li><details><summary><code>aarch64-linux</code></summary>

```shell
nix-store -r --add-root nixpkgs-pr-466553-aarch64-linux \
  --option binary-caches 'https://cache.nixos.org/ https://acidbong.cachix.org' \
  --option trusted-public-keys '
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  acidbong.cachix.org-1:ycS7jRCupEvvQRTlKcB4mjtCYUSlhioNQNwmrPs3E84=
  ' \
  /nix/store/93xs7wdscm4ll0x790z1n6i70rxnjfb6-pinsel-0-unstable-2022-03-27
```
</details></li>
</ul></details>


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinsel</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pinsel</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>pinsel</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>pinsel</summary>
<pre>Running phase: unpackPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/8d5zbk3acydcdp7mlf81lc3izpm1yi2z-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
no configure script, doing nothing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
build flags: SHELL=/nix/store/lbkni2ik42yc5hic017z8r6h5gn1gy04-bash-5.3p3/bin/bash INSTALLDIR=/nix/store/krhl8zh08bszzz8wmgr3khmzfc8lgsnb-pinsel-0-unstable-2022-03-27/bin
mkdir -p obj
gcc src/action.c -c -o obj/action.o -Wall -I/nix/store/27dw25yvi6qw4sz7sxkm0p5bsgmfdlyw-lua-5.2.4/include -I/nix/store/g8vi9vxak0zqb4c99xhsvsz8p71b3y2j-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/7i0nm45abd025dcclg1kni0hkas5hib2-at-spi2-core-2.58.1-dev/include/atk-1.0 -I/nix/store/axfhrz6zn5x2l2lbd0qls1nvafpaff9i-glib-2.86.1-dev/include/gio-unix-2.0 -I/nix/store/axfhrz6zn5x2l2lbd0qls1nvafpaff9i-glib-2.86.1-dev/include -I/nix/store/axfhrz6zn5x2l2lbd0qls1nvafpaff9i-glib-2.86.1-dev/include/glib-2.0 -I/nix/store/56x7dljphk1xv08h0dza9p2p9ndmy620-glib-2.86.1/lib/glib-2.0/include -I/nix/store/yrcscsnxar994hsgci9bh7cjvmjsp4g7-cairo-1.18.4-dev/include/cairo -I/nix/store/n8yhnz4pzj5m6k9x6s0yb91ppka9fwks-freetype-2.13.3-dev/include/freetype2 -I/nix/store/n8yhnz4pzj5m6k9x6s0yb91ppka9fwks-freetype-2.13.3-dev/include -I/nix/store/0ip5y9icm3ziy1iq19y6nk77hsxlb163-gdk-pixbuf-2.44.4-dev/include/gdk-pixbuf-2.0 -I/nix/store/pmm8cxg837vsn1z6r22casdmxq57dgnx-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/hmvykb39mbib1h0w3wvhzq7pyr8zvxkb-harfbuzz-12.1.0-dev/include/harfbuzz -L/nix/store/27dw25yvi6qw4sz7sxkm0p5bsgmfdlyw-lua-5.2.4/lib -L/nix/store/wvx4pksdkn0l8fxfm859y55bslssgiyi-gtk+3-3.24.51/lib -L/nix/store/gkfjxmwv2xrdv2iak04h1ldj44qg10ll-at-spi2-core-2.58.1/lib -L/nix/store/56x7dljphk1xv08h0dza9p2p9ndmy620-glib-2.86.1/lib -L/nix/store/z2vinhirimh75njgknwxxxz4irmcs1wj-cairo-1.18.4/lib -L/nix/store/dyd94xznh2rfjq714fm0j5ljjpfghllk-gdk-pixbuf-2.44.4/lib -L/nix/store/gxr2f0js7pb8ym09l97fgxz5rh7nlz3c-pango-1.57.0/lib -L/nix/store/wgdfscbdwgzxyirj4fm7scx0ym7yxfvd-harfbuzz-12.1.0/lib -lgtk-3 -lgdk-3 -lz -Wl,-framework,Cocoa -Wl,-framework,Carbon -Wl,-framework,CoreGraphics -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -llua -lgio-2.0 -lgobject-2.0 -lglib-2.0
/nix/store/lbkni2ik42yc5hic017z8r6h5gn1gy04-bash-5.3p3/bin/bash: line 1: gcc: command not found
make: *** [Makefile:24: obj/action.o] Error 127</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>pinsel</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>pinsel</summary>
<pre>Running phase: unpackPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/8d5zbk3acydcdp7mlf81lc3izpm1yi2z-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
no configure script, doing nothing
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
build flags: SHELL=/nix/store/p0k9r5h8qs7220xdbdihhfgzwjcly70x-bash-5.3p3/bin/bash INSTALLDIR=/nix/store/yhcadh7n79zdcagvp22jsrirj0cakzlz-pinsel-0-unstable-2022-03-27/bin
mkdir -p obj
gcc src/action.c -c -o obj/action.o -Wall -I/nix/store/js4iznnjf19qy3m5nihaz9x6ill5128a-lua-5.2.4/include -I/nix/store/r0ri4x7n9kp87ls49x4ssc06s5xv6mv2-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/sk8mal3v19sh5nh2g0yh5r2qwqa7c5qp-at-spi2-core-2.58.1-dev/include/atk-1.0 -I/nix/store/9l57268nckpacngwqz6ahadwzax22ja1-glib-2.86.1-dev/include/gio-unix-2.0 -I/nix/store/9l57268nckpacngwqz6ahadwzax22ja1-glib-2.86.1-dev/include -I/nix/store/9l57268nckpacngwqz6ahadwzax22ja1-glib-2.86.1-dev/include/glib-2.0 -I/nix/store/fhxjbdk0wmihyz8fjwmspswh6v9nkbhk-glib-2.86.1/lib/glib-2.0/include -I/nix/store/41p60wf0lz599g075ipl97zs5y8lf1pm-cairo-1.18.4-dev/include/cairo -I/nix/store/qi6gnavjkgnq06h70vlvbd4iy1kznr56-freetype-2.13.3-dev/include/freetype2 -I/nix/store/qi6gnavjkgnq06h70vlvbd4iy1kznr56-freetype-2.13.3-dev/include -I/nix/store/vln5snglwa320cff1ph82zaws0jc63dz-gdk-pixbuf-2.44.4-dev/include/gdk-pixbuf-2.0 -I/nix/store/l7x12g2mp48w7dm1x5rbbs9n0m7djbl8-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/7dxm7gv35qxqqzxfkk42gf0kxblm841q-harfbuzz-12.1.0-dev/include/harfbuzz -L/nix/store/js4iznnjf19qy3m5nihaz9x6ill5128a-lua-5.2.4/lib -L/nix/store/9sqz6h3wmlycd7jfx9imci1mcsp215li-gtk+3-3.24.51/lib -L/nix/store/k9vqx0p3c51b967fay876fkln971524l-at-spi2-core-2.58.1/lib -L/nix/store/fhxjbdk0wmihyz8fjwmspswh6v9nkbhk-glib-2.86.1/lib -L/nix/store/14j2m15qpjzpyh1j2drb9qwinlmhd9fc-cairo-1.18.4/lib -L/nix/store/x2a4v5zfs7p9y1lng0sw2fa8j5wv6nk0-gdk-pixbuf-2.44.4/lib -L/nix/store/63v3jc66cfvbl4dpmy72wq51y04ci42f-pango-1.57.0/lib -L/nix/store/j1wgx0pjgmgvmnsf4yhp91hn0pbw6ks5-harfbuzz-12.1.0/lib -lgtk-3 -lgdk-3 -lz -Wl,-framework,Cocoa -Wl,-framework,Carbon -Wl,-framework,CoreGraphics -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -llua -lgio-2.0 -lgobject-2.0 -lglib-2.0
/nix/store/p0k9r5h8qs7220xdbdihhfgzwjcly70x-bash-5.3p3/bin/bash: line 1: gcc: command not found
make: *** [Makefile:24: obj/action.o] Error 127</pre>
</details>
</details>

--- a/pkgs/by-name/pi/pinsel/package.nix
+++ b/pkgs/by-name/pi/pinsel/package.nix
@@ -42,6 +42,7 @@ stdenv.mkDerivation {
     homepage = "https://github.com/Nooo37/pinsel";
     license = licenses.mit;
     maintainers = with maintainers; [ lom ];
+    platforms = platforms.linux;
     mainProgram = "pinsel";
   };
 }

